Prog/Preservation_database.sas

```diff
@@ -627,21 +627,299 @@ set test;
   else COGregion=0;
   format COGregion COG. ;
 run;
+/*label variables*/
+data all;
+set all;
```

Comment:
you should not use the same name for both data sets - you overwriting all here - its better practice to use name the new data set something different. 

you can actually just move the label statements into the previous data step right before the run. there's no need to have a second datastep.  

please also change the revisions parameter to "New file."  this is a description of the type of process- if there is a change to this program later the revisions line might read some thing like "corrected indicator x" or "added code for y"